app/Platform/Enums/PlatformExecutionEnvironment.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Platform\Enums;
+
+enum PlatformExecutionEnvironment: string
+{
+    /** Windows, macOS, Linux */
+    case Desktop = 'desktop';
+
+    /** Android, iOS */
+    case Mobile = 'mobile';
+
+    /** Xbox, PlayStation, Switch */
+    case Console = 'console';
+
+    /** Raspberry Pi, similar SBCs */
+    case SingleBoard = 'single_board';
```

Comment:
Don't these usually run a flavor of Linux too?

Reply:
Not necessarily, for example: https://www.replayos.com/

Reply:
Granted - that's still a Linux distro. But it only runs on SBCs.

Reply:
We probably won't be able to detect if a user is hitting the downloads page directly with their Raspberry Pi. Raspian sets it user agent correctly, but it doesn't look like `jenssegers/agent` has any logic to sniff it out.

That's probably okay. There are still UI/UX benefits to setting the field. For example, we could use different iconography for SBC platforms, as well as visually separate them from Desktop, Mobile, etc.